api/v3/Payment.php

```diff
@@ -349,6 +364,12 @@ function _civicrm_api3_payment_get_spec(&$params) {
       'type' => CRM_Utils_Type::T_INT,
       'api.aliases' => ['payment_id', 'id'],
     ],
+    'is_payment' => [
```

Comment:
Cancel is implemented as a reversal - so I think this is changing the scope & intent of the cancel api in an unintended way